gregrickaby/viewer-for-reddit
# Feature/next app dir mantine v7

### Related Issue

Fixes #485

### Description

- Upgrade to Mantine v7
- Upgrade to Next.js 13 App Router

### Verification

How will a stakeholder test this PR?

1. View this on the automated Vercel link below
2. Verify functionality still works the same as production
